app/assets/stylesheets/components/_repo-list.scss

```diff
@@ -0,0 +1,67 @@
+.repo-list {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -$base-spacing;
+  padding-top: 1.25rem;
+}
+
+.repo-item {
+  background-color: darken($base-background-color, 5%);
+  border-radius: $base-border-radius;
+  box-shadow: 0 1px 0 lighten($base-background-color, 5%);
+  flex-basis: 300px;
```

Comment:
Do you want to keep this in 'rem's or 'em's like everything else, in case the `font-size` changes?